src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java

```diff
@@ -289,4 +289,19 @@ public void testLoadHeaderFileTwice() throws Exception {
         }
     }
 
+    @Test
+    public void testHeaderIsValidWithBlankLines() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(HeaderCheck.class);
+        checkConfig.addAttribute("headerFile", getPath("InputHeaderjava.blank-lines.header"));
+        checkConfig.addAttribute("ignoreLines", "");
```

Comment:
http://checkstyle.sourceforge.net/config_header.html#Header
ignoreLines are empty by default, why mention it here ?